media-sound/pasystray/pasystray-0.8.1.ebuild

```diff
@@ -37,6 +38,8 @@ src_prepare() {
 
 src_configure() {
 	econf \
+		$(use_enable X x11) \
+		$(use_enable appindicator appindicator) \
```

Comment:
what were the defaults here? and does that `X` use flag affect `x11-libs/libX11` or not?